include/secp256k1_musig.h

```diff
@@ -25,6 +25,8 @@ extern "C" {
  *        magic: Set during initialization in `pubkey_combine` to allow
  *               detecting an uninitialized object.
  *      pk_hash: The 32-byte hash of the original public keys
+ *    second_pk: Serialized x-coordinate of the second public key in the list. Is 0
```

Comment:
nit: "0" is not 100% clear for this type

Reply:
agree, fixed